lib/healthcheck/healthcheck-control.c

```diff
@@ -28,6 +28,25 @@
 #include "control/control-connection.h"
 #include "control/control-commands.h"
 #include "stats/stats-prometheus.h"
+#include "afinter.h"
+
+static inline void
+_append_internal_src_queue_metrics(AFInterMetrics metrics, GString *reply)
+{
+  gchar double_buf[G_ASCII_DTOSTR_BUF_SIZE];
+
+  gdouble queued = (gint) stats_counter_get(metrics.queued);
+  gsize queue_capacity = stats_counter_get(metrics.queue_capacity);
+
+  gdouble internal_queue_usage = 0;
```

Comment:
I think this should be 1, so if the queue_capacity is 0, we report full.

Reply:
Right, I'm fixing this.